Firmware/MotorControl/axis.cpp

```diff
@@ -33,6 +33,48 @@ Axis::Axis(int axis_num,
     update_watchdog_settings();
 }
 
+Axis::LockinConfig_t Axis::default_calibration() {
+    Axis::LockinConfig_t config;
+    config.current = 10.0f;           // [A]
+    config.ramp_time = 0.4f;          // [s]
+    config.ramp_distance = 1 * M_PI;  // [rad]
+    config.accel = 20.0f;     // [rad/s^2]
+    config.vel = 40.0f; // [rad/s]
+    config.finish_distance = 100.0f;  // [rad]
+    config.finish_on_vel = false;
+    config.finish_on_distance = true;
```

Comment:
If you are going to have finish on distance by default, you should increase the distance to be more than the max likely pole pairs we will ever see. Like 50pp? so like finish_distance = 2pi*50, something like that?